schema/meta/contract.json

```diff
@@ -16,7 +16,7 @@
       "type": "string",
       "minLength": 3,
       "maxLength": 24,
-      "pattern": "^[a-zA-Z0-9-_]{1,255}$"
+      "pattern": "^((?!-)[a-zA-Z0-9-]{0,62}[a-zA-Z0-9])$"
```

Comment:
Please, increase number of characters to 255 by using `{0,254}` instead of `{0,62}`

Reply:
You should change it in other places as well.